api/base/settings/defaults.py

```diff
@@ -236,13 +236,14 @@
 USE_TZ = True
 
 # https://django-storages.readthedocs.io/en/latest/backends/gcloud.html
-
 if os.environ.get('GOOGLE_APPLICATION_CREDENTIALS', False):
     # Required to interact with Google Cloud Storage
     DEFAULT_FILE_STORAGE = 'storages.backends.gcloud.GoogleCloudStorage'
     GS_BUCKET_NAME = os.environ.get('GS_BUCKET_NAME', 'cos-osf-stage-osf-cdn')
     GS_FILE_OVERWRITE = os.environ.get('GS_FILE_OVERWRITE', False)
-
+else:
```

Comment:
This should perhaps check for `DEV_MODE` or `DEBUG_MODE`, erroring if not. We don't want deployed environments, especially prod, to use this because of a devops oversight.